plugins/woo-ai/src/hooks/useTinyEditor.ts

```diff
@@ -0,0 +1,33 @@
+type TinyContent = {
+	getContent: () => string;
+	setContent: ( str: string ) => void;
+};
+
+declare const tinymce: { get: ( str: string ) => TinyContent };
+
+export const useTinyEditor = () => {
+	const getTinyContentObject = () =>
+		typeof tinymce === 'object' ? tinymce.get( 'content' ) : null;
+
+	const setTinyContent = ( str: string ) => {
```

Comment:
Might be good to do an initial string length check to determine that an empty string isn't being passed.

```if ( ! str.length ) return;```